data/card_catalogue/card-manipulation.json

```diff
@@ -131,26 +131,13 @@
   },
   {
     "title": "Extemporize",
-    "upgrade": "Helter-Skelter",
+    "upgrade": null,
     "tier": "red",
     "details": {
       "rules": {
         "active": "Draw 5 greens, face-up. Cast up to 3 and discard the rest.",
         "passive": null
       },
-      "flavour": "People will forget what you said, people will forget what you did, but people will never forget when you pulled out a fish in a fistfight.",
-      "image": "res/art/illustration/Extemporize.png"
-    }
-  },
-  {
-    "title": "Helter-Skelter",
-    "upgrade": null,
-    "tier": "gold",
-    "details": {
-      "rules": {
-        "active": "Draw 5 reds, face-up. Cast up to 3 and discard the rest.",
-        "passive": null
-      },
```

Comment:
Doesn't belong at gold with recent card design heuristics.

Reused the art we had, but not the name since Manson references are probably not great if you aren't the beatles